src/sentry/receivers/rules.py

```diff
@@ -12,13 +12,9 @@
 }
 
 
-def create_default_rules(instance, created=True, RuleModel=Rule, **kwargs):
-    if not created:
-        return
+def create_default_rules(project, default_rules=True, RuleModel=Rule, **kwargs):
+    if default_rules:
+        RuleModel.objects.create(project=project, label=DEFAULT_RULE_LABEL, data=DEFAULT_RULE_DATA)
 
-    RuleModel.objects.create(project=instance, label=DEFAULT_RULE_LABEL, data=DEFAULT_RULE_DATA)
 
-
-post_save.connect(
-    create_default_rules, sender=Project, dispatch_uid="create_default_rules", weak=False
-)
+project_created.connect(create_default_rules, dispatch_uid="create_default_rules", weak=False)
```

Comment:
This change feels dangerous, since we don't fire this signal in a consistent place. One alternative that feels a little hacky, but would be safer, could be to just set a `_create_default_rules` attribute on the project, and look for it in `post_save`. Something like

```
project = Project(<params>)
project._default_rules = default_rules
project.save()
```

Reply:
I think thats acceptable, because technically speaking we'd like to avoid these kinds of side effects when creating test fixtures. That means we can choose to fire the signal or not.

We do after all only have a couple of entry points for this, and if project_created doesnt fire then some other things will break.

Reply:
The alternative is something equally failure-capable: Project.create_project()

Reply:
Aside, I need to verify we dont hook this signal in unexpected ways in saas (to where user has to be an actual User instance) before we can merge.